types/async-lock/index.d.ts

```diff
@@ -1,6 +1,8 @@
-// Type definitions for async-lock
+// Type definitions for async-lock 1.1.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
ok. I've enabled all linting rules: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23340/commits/2623588675a7b5ea2c3845edb8af7df9b13840c0